cmd/server.go

```diff
@@ -202,5 +208,15 @@ func loadAndVerifyServerConfig(cmd *cobra.Command) (*config.ServerConfig, error)
 		}
 	}
 
+	if conf.DataDir == "" {
+		cmd.Printf(color.HiYellowString("[WARN] TiCDC server data-dir is not set. " +
+			"Please use `cdc server --data-dir` to start the cdc server if possible.\n"))
+	}
+
+	if serverConfig.Sorter.SortDir != config.DefaultSortDir {
+		cmd.Printf(color.HiYellowString("[WARN] --sort-dir is deprecated in server settings. " +
```

Comment:
You wrote `conf.Sorter.SortDir = config.DefaultSortDir`. So when will this be run?

Reply:
we should put it when trying to set conf.Sort.SortDir